loguru.cpp

```diff
@@ -1015,8 +1015,25 @@ namespace loguru
 #if LOGURU_WINTHREADS
 	char* get_thread_name_win32()
```

Comment:
This function should be called `get_thread_name_buffer` or something like that. The current name confused you into thinking its purpose it to fetch the thread name, which is in fact not the purpose of it at all.

The purpose of this function is to just return a pointer to a thread-local storage for the thread name that `set_thread_name` and `get_thread_name` then use